### PR TITLE
Create Avatar component

### PR DESCRIPTION
**Pull request by JonathanYesh279**

### CODE REVIEW

File: src/components/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+function getUserInitials(name) {
+  if (!name) return 'GU';
+  return name
+    .split(' ')
+    .map((part) => part[0])
+    .join('')
+    .toUpperCase()
+    .slice(0, 2);
+}
```
Comment (tomercarolla) [inline src/components/Avatar.jsx:11]: move to the end of the file

Comment (tomercarolla) [reply src/components/Avatar.jsx:11]: also if not have name return null

File: src/components/Avatar.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <AvatarWrapper
+      $imageUrl={imageUrl}
+      size={size}
+      fontSize={fontSize}
+      title={name || 'User'}
+      role="img"
+      aria-label={name || 'User avatar'}
+    >
+      {imageUrl ? null : initials}
```
Comment (tomercarolla) [inline src/components/Avatar.jsx:29]: if you have imageUrl you return null?
better to do an img tag and add the imageUrl there.
not forget if not have image, there is an initial letters and background-color

Comment (tomercarolla) [reply src/components/Avatar.jsx:29]: you dont pass the bgColor prop

File: src/components/Avatar.jsx
Comment (tomercarolla) [inline src/components/Avatar.jsx:1]: the avatar can be in multiple places, move to ui/avatar, add indes.jsx and export it like the other components

File: src/components/Avatar.jsx (hunk 1 above)
Comment (JonathanYesh279) [reply src/components/Avatar.jsx:11]: why this function should be at the end of the file? i think it makes more sense to read the function and then the actual use of it inside the component, right?

**Review by tomercarolla — APPROVED** [review]

**Pull request merged**